dateutil/test/test_parser.py

```diff
@@ -833,7 +834,7 @@ def test_on_era(self):
     def test_extraneous_year(self):
         # This was found in the wild at insidertrading.org
         dstr = "2011 MARTIN CHILDREN'S IRREVOCABLE TRUST u/a/d NOVEMBER 7, 2012"
-        res = parse(dstr, fuzzy_with_tokens=True)
+        res, _ = parse(dstr, fuzzy_with_tokens=True)
```

Comment:
If we're not using the tokens, should we use `fuzzy=True` instead? Or is the bug only triggered when you capture the tokens?

Reply:
You are of course correct. I only noticed these by chance and quickly made the smallest change necessary to make them pass without even thinking about it.